packages/react/src/date-picker/utils.test.ts

```diff
@@ -47,57 +46,6 @@ describe('constrainDate', () => {
 	});
 });
 
-describe('getValidDateRange', () => {
```

Comment:
`getValidDateRange` is only ever used inside of `date-range-picker` so it does make sense for this code to live here. This is just a hangover from when date-picker and date-range-picker lived in the same folder.